telegram/ext/regexhandler.py

```diff
@@ -50,12 +50,16 @@ class RegexHandler(Handler):
             the callback function.
         pass_chat_data (:obj:`bool`): Optional. Determines whether ``chat_data`` will be passed to
             the callback function.
+        use_context (:obj:`bool`): Optional. Determines whether all `pass_` arguments will be
```

Comment:
I think we can deprecate regexhandler in total. See my note about filters.

Reply:
I suggest the RegexHandler be a convenience wrapper over MessageHandler.